src/utils.ts

```diff
@@ -93,3 +93,7 @@ export const checkIfEnvironmentVariableIsOmitted = (key: string): boolean => {
 
     return omitEnvironmentVariablesWithPrefix.some((prefix: string) => key.toUpperCase().startsWith(prefix));
 }
+
+export const escapeDoubleQuote = (value: string): string => {
+    return value.replace(/"/g, '\\"');

```

Comment:
Regex replacement can be much slower than a simply substring replace in some browsers. Alternatively, we can consider

```suggestion
    return value.replaceAll('"', '\\"');
```

See https://khef.co/2019-javascript-string-replace-regex-vs-substring/